src/content/docs/query-your-data/nrql-new-relic-query-language/nrql-query-tutorials/query-infrastructure-dimensional-metrics-nrql.mdx

```diff
@@ -9,7 +9,7 @@ metaDescription: 'In New Relic, you can query dimensional metrics for Infrastruc
 
 import namingConvention from 'images/naming-convention.png'
 
-<Callout title="BETA FEATURE">
+<Callout title="PREVIEW FEATURE">
   This feature is currently in beta.
```

Comment:
Change "This feature is currently in beta." to "This feature is currently in preview".